Exiled.API/Features/Player.cs

```diff
@@ -2212,6 +2216,91 @@ public void Reconnect(ushort newPort = 0, float delay = 5, bool reconnect = true
         /// <inheritdoc cref="Map.GetNearCameras(Vector3, float)"/>
         public IEnumerable<Camera> GetNearCameras(float toleration = 15f) => Map.GetNearCameras(Position, toleration);
 
+        /// <summary>
+        /// Teleports the player to the given <see cref="Vector3"/> coordinates.
+        /// </summary>
+        /// <param name="position">The <see cref="Vector3"/> coordinates to move the player to.</param>
+        public void Teleport(Vector3 position) => Position = position;
```

Comment:
Not sure I see the use in having another method to access set_Position

Reply:
I don't either but if we're ganna add teleport methods we might aswell have this? 
I think
Teleport(door.Position + vector3.up)
looks better than
Position = door.Position + Vector3.up
but that's just me, personally. We could remove this teleport method.